src/notation/inotationinteraction.h

```diff
@@ -82,6 +82,257 @@ class INotationInteraction
     virtual INotationSelectionPtr selection() const = 0;
     virtual void clearSelection() = 0;
     virtual muse::async::Notification selectionChanged() const = 0;
+    virtual muse::async::Notification playbackNotesChanged() const 
```

Comment:
Methods in interface classes should always be _pure_ virtual, i.e. with `= 0` instead of an implementation.